requirements.yml

```diff
@@ -7,3 +7,6 @@
   src: MaximeThoonsen.php5-xdebug
 - name: swapfile
   src: kamaln7.swapfile
+- name: mailhog
+  src: https://github.com/Berdir/ansible-role-mailhog
```

Comment:
I'm using the fork of the pull request until https://github.com/geerlingguy/ansible-role-mailhog/pull/2 in merged, because the role install ssmtp by default and we do it too, so it would conflict.